bin/vaultclient

```diff
@@ -104,20 +104,27 @@ program
     .option('--name <NAME>')
     .option('--email <EMAIL>')
     .option('--quota <Quota>', 'Maximum quota for the account', parseInt)
+    .option('--accountid <ExternalAccountID>')
     .action(action.bind(null, 'create-account', (client, args) => {
         client.createAccount(
             args.name, {
-                email: args.email,
-                quota: args.quota || null
-            },
+            email: args.email,
+            quota: args.quota || null,
+            externalAccountId: args.accountid || null
+        },
             handleVaultResponse);
     }));
 
 program
     .command('generate-account-access-key')
     .option('--name <NAME>')
+    .option('--accesskey <ExternalAccessKey>')
```

Comment:
Because time may lack, I'm not against keeping this change, but I would think better to have a separate command `import-account-access-key` since their respective purpose is rather different.

Reply:
Yeah I agree! due to lack of time and effort and tests required. I integrated this to the existing code. In the future this should be seperate.